db/schema.rb

```diff
@@ -1687,7 +1547,8 @@
     t.boolean "stem_indicator", default: false
     t.string "campus_type"
     t.string "parent_facility_code_id"
-    t.index ["facility_code"], name: "index_weams_on_facility_code", unique: true
+    t.integer "csv_row"
+    t.index ["facility_code"], name: "index_weams_on_facility_code"
```

Comment:
Looks like this sneaked in from the other PR.  Remove please.

Reply:
removed